composer.json

```diff
@@ -41,13 +41,13 @@
     },
     "scripts": {
         "check-code": [
-            "./vendor/bin/phpcs --standard=phpcs.xml -snp --encoding=utf-8 src/ tests/ --report-width=150"
+            "\"vendor/bin/phpcs\" --standard=phpcs.xml -snp --encoding=utf-8 src/ tests/ --report-width=150"
```

Comment:
@jacklul Have you also tried it without the escaped double-quotes?
Just: `"vendor/bin/phpcs ..."`

Reply:
Yep! Doesn't work either!

```
>composer check-code
> vendor/bin/phpcs --standard=phpcs.xml -snp --encoding=utf-8 src/ tests/ --report-width=150
'vendor' is not recognized as an internal or external command,
operable program or batch file.
Script vendor/bin/phpcs --standard=phpcs.xml -snp --encoding=utf-8 src/ tests/ --report-width=150 handling the check-code event returned with error code 1
```

Reply:
Uff, windows... 😁